### PR TITLE
fixed issue #864

### PR DESCRIPTION
**Pull request by yichaoS**

### CODE REVIEW

File: portal/src/main/webapp/js/src/cross-cancer-plotly-plots.js [hunk 1]
```diff
@@ -211,7 +211,18 @@ var ccPlots = (function (Plotly, _, $) {
                                 study_ids = _.filter(study_ids, function(study_id) { return study_id.indexOf(_tmp_target_study_obj[0].id) === -1; });
                                 mrna_profiles = _.filter(mrna_profiles, function(mrna_profile) { return mrna_profile.indexOf(_tmp_target_study_obj[0].id) === -1; });
                             }
-
+    
+                            //exclude stomach
+                            var _tmp_target_study_obj = _.filter(study_meta, function(obj) { return obj.name.indexOf("Stomach") !== -1 });
```
Comment (jjgao) [inline portal/src/main/webapp/js/src/cross-cancer-plotly-plots.js]: It seems to be a lot of code to exclude a study. Can we put all the code in one function to get a list of studies to be used for this analysis, eg. TCGA provisional studies excluding STAD and ESCA?

**Comment by yichaoS** [comment]
Definitely. Sorry didn't think much for this change.

**Comment by yichaoS** [comment]
screen shot 2016-03-03 at 6 05 45 pm

**Comment by yichaoS** [comment]
screen shot 2016-03-03 at 6 05 51 pm

**Pull request merged**